test

```diff
@@ -214,6 +214,15 @@ function dep_pass {
 	fi
 }
 
+function errcheck_pass {
+	echo "Running errcheck tests..."
+        export PATH="$PATH:$GOPATH/bin"
+        if ! which errcheck > /dev/null ; then
+                go get github.com/kisielk/errcheck || { echo "Failed to install 'errcheck'"; exit 255; }
```

Comment:
the `go get` should be in `.travis.yml`; the pass itself probably belongs in `fmt_pass` since that's where the other static checks live